src/Client.php

```diff
@@ -19,23 +19,46 @@ public function __construct(array $config = [])
 
         if (!empty($config['access_token'])) {
             $config['headers'] = [
-                'Authorization' => 'Bearer ' . $config['access_token']
+                'Authorization' => 'Bearer '.$config['access_token'],
             ];
         }
         parent::__construct($config);
     }
 
     /**
-     * Get default configuration to apply the client
+     * Get default configuration to apply the client.
+     *
      * @return array
      */
     protected function getDefaultClientConfig()
     {
         return [
-            'base_uri' => 'https://api.creads-partners.com/v1'
+            'base_uri' => 'https://api.creads-partners.com/v1',
         ];
     }
 
+    public function put($uri = '', $body = [])
```

Comment:
I figured, as it doesnt raise any security issue, let the user take a 404 error or any other and don't pre-block possibilities to keep the library generic.
But I agree PUT/POST methods dont need a default value, i'll change that.

Okay for the options array